dll/win32/shell32/shlexec.cpp

```diff
@@ -28,6 +28,8 @@ WINE_DEFAULT_DEBUG_CHANNEL(exec);
 static const WCHAR wszOpen[] = L"open";
 static const WCHAR wszExe[] = L".exe";
 static const WCHAR wszCom[] = L".com";
+static const WCHAR wszBat[] = L".bat";
+static const WCHAR wszCmd[] = L".cmd";
```

Comment:
Shouldn't all this be covered by the `PathResolveW` API @katahiromz finally implemented recently?

Reply:
I tried working with the PathResolveW function, but I was not able to easily create workable code myself with it. Can you attach a patch in the original issue [CORE-17612](https://jira.reactos.org/browse/CORE-17612) that we can discuss? Thanks.